spec/struct.dd

```diff
@@ -141,9 +141,9 @@ struct S;
 union U;
 ---
 
-	$(P The members are completely hidden to the user, and so the only operations
-	on those types are ones that do not require any knowledge of the contents
-	of those types. For example:)
+        $(P The members are completely hidden to the user, and so the only operations
```

Comment:
No. Trivial test: if you can select individual spaces (here, on this webpage even), then they're not hard tabs.